### PR TITLE
#68|Parmeterize number of recent posts in index.html

### PR DESCRIPTION
**Pull request by DeveloperDavo**

### CODE REVIEW

**Comment by DeveloperDavo** [comment]
Let me know if you would also like me to parameterize the number of more posts as well. Something like:
```
[params]
  more_posts_number = 4  
```

File: layouts/index.html [hunk 1]
```diff
@@ -20,20 +20,22 @@ <h1 class="flex-none">
           </h1>
         {{ end }}
 
+      {{ $n_posts := .Site.Params.recent_posts_number | default 3 }}
```
Comment (budparr) [inline layouts/index.html]: This should be `$.Param "recent_posts_number"` [Param](https://gohugo.io/functions/param) allows users to set the variable at the page level or config.

**Comment by budparr** [comment]
Thank you, @DeveloperDavo I just had the one comment.

**Comment by budparr** [comment]
Thanks again!

**Pull request merged**